doc/sphinx/language/core/records.rst

```diff
@@ -39,6 +39,12 @@ expressions. In this sense, the :cmd:`Record` construction allows defining
    order of the fields is important. :n:`@binder` parameters may be applied to the record as a whole
    or to individual fields.
 
+   :n:`{? > }`
+     If specific, the constructor name is automatically declared as
+     a coercion from the class of the last field type to the record name
+     (this may fail if the uniform inheritance condition is not
+     satisfied).
```

Comment:
It's good that this is described here, but it should also be briefly mentioned in the Coercions chapter.

Reply:
Think that's covered now.